src/commands/moderation/ban.ts

```diff
@@ -0,0 +1,80 @@
+import { sendMessage, ban, highestRole, sendDirectMessage, Member, botID } from "./../../../deps.ts";
+import { Embed } from "./../../utils/Embed.ts";
+import { createCommand, sendEmbed } from "./../../utils/helpers.ts";
+
+createCommand({
+  name: `ban`,
+  guildOnly: true,
+  arguments: [
+    {
+      name: "member",
+      type: "member",
+      missing: (message) => {
+        return sendMessage(message.channelID, "User not found!");
+      },
+    },
+    {
+      name: "days",
+      type: "number",
+      defaultValue: 0,
+    },
+    {
+      name: "reason",
+      type: "...string",
+      defaultValue: "No reason given",
+    },
+  ],
+  userServerPermissions: ["BAN_MEMBERS"],
+  botServerPermissions: ["BAN_MEMBERS"],
+  botChannelPermissions: ["VIEW_CHANNEL", "SEND_MESSAGES", "EMBED_LINKS"],
+  execute: async (message, args: BanArgs) => {
+    try {
+      const botHighestRolePosition = (await highestRole(message.guildID, botID))!.position;
+      const memberHighestRolePositon = (await highestRole(message.guildID, args.member.id))!.position;
+      const authorHighestRolePosition = (await highestRole(message.guildID, message.author.id))!.position;
+
+      if (memberHighestRolePositon >= botHighestRolePosition || memberHighestRolePositon >= authorHighestRolePosition) {
```

Comment:
This will not always work. You need to account for other edge cases. There is a built in function `higherRolePosition` in discordeno that checks if the role is higher. This should be used instead of comparing position like this.